### PR TITLE
2.17.x

### PR DESCRIPTION
**Pull request by kneth**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
+2.16.0 Release notes
+=============================================================
+### Breaking changes
+* No API breaking changes but the sync protocol is updated to version 26. You will need to upgrade your Realm Object Server to at least version 3.11.0 or use [Realm Cloud](https://cloud.realm.io). If you are using an old Realm Object Server, you will seee error message `Wrong protocol version in Sync HTTP request, client protocol version = 25, server protocol version = 24`. See https://docs.realm.io/platform/using-synced-realms/troubleshoot/version-compatibilities for more information.
```
Comment (bmunkholm) [inline CHANGELOG.md]: @kneth I believe I reformulated this once and placed the message under Compatibility?

Comment (bmunkholm) [reply CHANGELOG.md]: Found it. Please use the changelog from here: https://github.com/realm/realm-js/pull/2025 (except for what's no longer included.

Comment (kneth) [reply CHANGELOG.md]: I might have forgot to do a `git pull` when renaming the branch. I'll update this PR today.

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,40 @@ x.x.x Release notes (yyyy-MM-dd)
...
 
+2.17.0 Release notes (2018-9-25)
+=============================================================
+### Enhancements
+* When using a synced Realm where you only receive updates from the server and never update the realm locally, the client will now report download progress to the server such that it can compact files on the server. This enhancement doesn't affect the client.
+
+### Compatibility
+* File format: ver. 7 (upgrades automatically from previous formats)
+* **Realm Object Server: 3.11.0 or later.**
+You will need to upgrade your Realm Object Server to at least version 3.11.0 or use [Realm Cloud](https://cloud.realm.io).
```
Comment (bmunkholm) [inline CHANGELOG.md]: This is now written twice - (also above)

**Pull request merged**